tests/yspryn/test_fraction_hw9.py

```diff
@@ -31,7 +32,6 @@ def test_fractions():
     # # add
     assert Fraction(2, 4) + Fraction(1, 3) == Fraction(5, 6)
     assert Fraction(1, 2) + 2 == Fraction(5, 2)
-    assert 2 + Fraction(1, 2) == Fraction(5, 2)
```

Comment:
why removed test

Reply:
you should also cover with tests right side operators

__radd__
__rsub__.  etc.

Reply:
I believe I've removed it accidentally, it was passed for sure